src/_includes/common/configuration-basics.md

```diff
@@ -0,0 +1,184 @@
+{% comment %}
+Guideline: This page is comprehensive; it is stored in the common folder, nested under _includes/common. To use, 
+
+1. Add a folder with the name of the platform you are documenting to the _documentation/sdks structure (for example, _documentation/sdks/javascript) 
+2. Create a copy of "basics.md" file in _documentation/sdks/<platform-name> 
+3. Create the defined `include` statements and add them to the configuration-basics.md file
+4. Note that each of these terms is wrapped in an "if/then" statement; this is because not all of these options are included for every SDK. For those terms you do not wish to include, add the command `hide_<option>=true` in the SDK-specific page.
+
+If you have questions, please ask Fiona or Daniel. 
+
+**The objective for this page is that a developer can view easily the configuration options for the SDK; each page _must_ have a description that includes a summary of what the page provides to the developer. Simply linking the page is insufficient.**
+{% endcomment %}
+
+The {{ include.sdk_name }} SDK is configurable using a variety of options. These options are set when the SDK is first initialized, passed to the `init()` as an object:
+
+{{ include.config_basic_content }}
+
+These options can be read from an environment variable or your `~/.sentryclirc` file (`SENTRY_DSN`, `SENTRY_ENVIRONMENT`, `SENTRY_RELEASE`) are read automatically.
+
+{% if include.hide_dsn == nil %}
+`dsn` 
+
+The *DSN* tells the SDK where to send the events. If this value is not provided, the SDK will try to read it from the `SENTRY_DSN` environment variable. If that variable also does not exist, the SDK will not send any events.
+
+In runtimes without a process environment (such as the browser SDK), the fallback of reading from the environment variable does not apply.
+{% endif %}
+{% if include.hide_debug == nil %}
+`debug`
+
+Turns debug mode on or off. If debug is enabled, the SDK will attempt to print out useful debugging information if something goes wrong with sending the event. The default is always `false`. It’s generally not recommended to turn it on in production, though turning debug mode on will not cause any safety concerns.
+{% endif %}
+{% if include.hide_release == nil %}
+`release`
+
+Sets the release. The SDK will try to automatically configure a release out of the box if it's provided by `window.SENTRY_RELEASE`, but it's best to set it manually to guarantee that the release is in sync with your deploy integrations or source map uploads.
```

Comment:
`window.SENTRY_RELEASE` is for @sentry/browser only. I'd use the same sentence that we have in DSN section ("read it from the`SENTRY_DSN` environment variable").

Reply:
Thanks.